Gemfile

```diff
@@ -10,18 +10,19 @@ source "https://rubygems.org"
 # Happy Jekylling!
 gem "sassc", "~> 2.0.0"
 
-gem "jekyll", "~> 3.8.5"
+gem "jekyll", "~> 3.7.3"
```

Comment:
Does gh-pages require the downgrade to the Jekyll version?

Reply:
so yeah, you can run $ bundle exec github-pages versions to see all the dependencies it will enforce, which encludes jekyll 3.7.3